composer.json

```diff
@@ -11,8 +11,8 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "laravel/framework": "^5.4 || ^6.0"
+        "php": "^7.1 || ^7.2.5",
```

Comment:
Why this? `^7.1` should already resolve to `7.1 <= 8.0`, which includes 7.2.

Reply:
oops sorry sorry sorry. my mistake i am reverting it back.